virtual_rainforest/models/animals/constants.py

```diff
@@ -1,62 +1,69 @@
-FAT_MASS_TERMS = ScalingTerms(1.19, 0.02)
-
-"""Scaling of mammalian herbivore muscle mass (citation from Rallings).[g]"""
-MUSCLE_MASS_TERMS = ScalingTerms(1.0, 0.38)
-
-"""Mammalian maximum intake rate (g/min) from (Shipley 1994)."""
-INTAKE_RATE_TERMS = ScalingTerms(0.71, 0.63)
-
-
-@dataclass
-class Term:
-    """Dataclass for handling fixed parameter values.
-
-    In the future, this class will be significantly expanded to take a suite of animal
-    cohort traits, environmental conditions, and/or food traits as input to determine
-    the final required parameter value.
-
-    """
-
-    value: float
-    """Contains the float value of the parameter"""
-
-
-"""The energy of a unit mass of mammal meat (check citation from Rallings). [J/g]"""
-MEAT_ENERGY = Term(7000.0)
+The near-future intention is to rework the relationship between these constants and the
+AnimalCohort objects in which they are used such that there is a FunctionalType class
+in-between them. This class will hold the specific scaling, rate, and conversion
+parameters required for determining the function of a specific AnimalCohort and will
+avoid frequent searches through this constants file for values.
+"""  # noqa: D205, D415
 
-"""Toy value expressing how much of a consumed food is absorbed as energy. [unitless]"""
-CONVERSION_EFFICIENCY = Term(0.1)
 
-"""Temporary realistic plant food value: Alfalfa ¬ 18,200,000 J/kg DM."""
-PLANT_ENERGY_DENSITY = Term(18200000.0)
+ENDOTHERM_METABOLIC_RATE_TERMS: dict[str, tuple[float, float]] = {
+    "mammal": (0.75, 0.047),
+    # Mammalian herbivore population density, observed allometry (Damuth 1987). [kg]
+    "bird": (0.75, 0.05),
+    # Toy values.
+}
+
+DAMUTHS_LAW_TERMS: dict[str, dict[str, tuple[float, float]]] = {
+    "mammal": {
+        "herbivore": (-0.75, 4.23),
+        # Mammalian herbivore population density, observed allometry (Damuth 1987). [kg]
+        "carnivore": (-0.75, 1.00),
+        # Toy values.
+    },
+    "bird": {
+        "herbivore": (-0.75, 5.00),
+        # Toy values.
+        "carnivore": (-0.75, 2.00),
+        # Toy values.
+    },
+}
+
+FAT_MASS_TERMS: dict[str, tuple[float, float]] = {
+    "mammal": (1.19, 0.02),
+    # Scaling of mammalian herbivore fat mass (citation from Rallings). [g]
+    "bird": (1.19, 0.05),
+    # Toy Values
+}
+
+MUSCLE_MASS_TERMS: dict[str, tuple[float, float]] = {
+    "mammal": (1.0, 0.38),
+    # Scaling of mammalian herbivore muscle mass (citation from Rallings).[g]
+    "bird": (1.0, 0.40),
+    # Toy Values
+}
+
+INTAKE_RATE_TERMS: dict[str, tuple[float, float]] = {
+    "mammal": (0.71, 0.63),
+    # Mammalian maximum intake rate (g/min) from (Shipley 1994).
+    "bird": (0.7, 0.50),
+    # Toy Values
+}
+
+
+ENERGY_DENSITY: dict[str, float] = {
+    "meat": 7000.0,
+    # The energy of a unit mass of mammal meat (check citation from Rallings). [J/g]
+    "plant": 18200000.0
+    # Temporary realistic plant food value: Alfalfa ¬ 18,200,000 J/kg DM.
```

Comment:
It is in constants because it is the first of a variety of different plant energy densities for different plant parts/types. I can put it in it's own category of constants if that helps with the units issue but I'd rather not put 4-5 different energy density constants inside the PlantCommunity. I know it's all temporary but I'll soon be needing to check that herbivores are foraging successfully over multiple different plant resources.